gallery/how_to/deploy_models/deploy_model_on_adreno.py

```diff
@@ -53,11 +53,17 @@
 #
 #   adb devices
 #
+# Set the android device to use
```

Comment:
I would propose to clarify that specifying `ANDROID_SERIAL` is necessary only if you have several Android devices.
```suggestion
# Set the android device to use, if you have several devices connected to your computer.
```